gulpfile.js

```diff
@@ -171,7 +171,6 @@ gulp.task('vf-css', function(done) {
             return 'Problem at file: ' + error.message;
           })
         )
-        .pipe(autoprefixer(autoprefixerOptions))
```

Comment:
Was this intentional? I know we've discussed that dropping autoprefixer would likely be a good optimisation, but it's still in the package.json dependencies.

Reply:
No. Unfortunately you can swap a PR to a draft PR. Which this still is.

Reply:
I'll put this back in and push